cocotb/__init__.py

```diff
@@ -112,16 +112,17 @@ def mem_debug(port):
 
 
 def _initialise_testbench(root_name):
-    """
+    """Initialize testbench.
+
     This function is called after the simulator has elaborated all
     entities and is ready to run the test.
 
-    The test must be defined by the environment variables
-        MODULE
-        TESTCASE
+    The test must be defined by the environment variables:
+    - :envvar:MODULE`
+    - :envvar:TESTCASE`
```

Comment:
Perhaps just flow these into the sentence, rather than as a list

Reply:
Makes sense, done.